helm/hpcc/templates/dafilesrv.yaml

```diff
@@ -101,5 +102,45 @@ kind: ConfigMap
 {{ include "hpcc.addService" ( dict "root" $ "name" .name "service" $service "selector" .name "defaultPort" 7600 ) }}
 ---
 {{ include "hpcc.addCertificate" (dict "root" $ "name" .name "service" .service "component" "dafilesrv" "visibility" .service.visibility) }}
+{{- if eq .application "spray" -}}
```

Comment:
A comment above this line marking the start of where the generation of the Network policy may help the readability a bit.